### ISSUE TITLE
test issue before posting to GO issue tracker

### ISSUE THREAD

**Issue by andrewsu** [SuLab/ChlamBase.org#45]
This issue shows draft text to post on the GO issue tracker for input.  Comments welcome, all but particularly @khybiske 

-----------

In the context of a wikidata-backed project called Chlambase, we would like to annotate the localization of a set of Chlamydia proteins to various compartments in the host and in the inclusion body (which is the nucleus-sized structure indicated by the red arrow):

![2018-04-02_15-24-13](https://user-images.githubusercontent.com/2635409/38282391-ae7c2e62-3764-11e8-959c-07b48ccf01db.png)

This table summarizes the localizations that our domain expert (@khybiske) has identified with example genes and references.  We've also listed our best guess GO term with our confidence level in that guess.

Localization | Example Chlamydia Gene ID | Supporting PMID | Best guess GO | confidence level
-- | -- | -- | -- | --
host actin cytoskeleton | CT456 | [15199184 ](https://www.ncbi.nlm.nih.gov/pubmed/15199184 )| [GO_0005884 ](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0005884 )| low
inclusion lumen | CT049 | [18591235 ](https://www.ncbi.nlm.nih.gov/pubmed/18591235)| [GO_0020004 ](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0020004 )| low
inclusion membrane | CT005 | [26416906 ](https://www.ncbi.nlm.nih.gov/pubmed/26416906 )| [GO_0020005 ](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0020005 )| low
host cell nucleus | CT737 | [20657819 ](https://www.ncbi.nlm.nih.gov/pubmed/20657819 )| [GO_0042025 ](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0042025 )| high
host cell cytosol | CT858 | [20522495 ](https://www.ncbi.nlm.nih.gov/pubmed/20522495 )| [GO_0044164 ](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0044164 )| high
host lipid droplets | CT163 | [16920627 ](https://www.ncbi.nlm.nih.gov/pubmed/16920627)| [GO_0044186 ](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0044186 )| high
host cell plasma membrane | CT867 | [16824101 ](https://www.ncbi.nlm.nih.gov/pubmed/16824101 )| [GO_0020002 ](https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0020002 )| high

Does anyone have input on these mappings, particularly if there are better mappings or cases where new GO terms could/should be created?

**Comment by khybiske** [comment]
I agree with our low confidence for the inclusion-related GO terms. Why a low confidence for the actin one?

**Comment by khybiske** [comment]
@andrewsu I don't know if going outside GO is acceptable, but BTO:0001172 and BTO:0000377 are perfect matches for the 2 RB/EB developmental forms of Chlamydia that we have depicted in our image. Ideally, would be great to capture these annotations as we have that data for many proteins.

**Comment by andrewsu** [comment]
Regarding actin, the reason I don't think this is appropriate is because if a chlamydia gene were annotated to this term, it would imply that it localizes to the an actin filament in chlamydia.  I think ultimately we'll need "host actin filament" term to be created...

Going outside GO is totally acceptable, and those BTO look great for RB/EB.  It might be worth you searching a bit in https://www.ebi.ac.uk/ols to see if you can't find something for inclusion membrane/lumen.
